dsm/structs.go

```diff
@@ -90,13 +90,30 @@ type AzureSobject struct {
 	Creator     DSMCreator
 	Kid         string
 	Obj_type    string
-	//	Custom_metadata AWSSobjectCustomMetadata
+	Custom_metadata AzureSobjectCustomMetadata
 	Enabled bool
 	State   string
-	//	External        AWSSobjectExternal
+	External        AzureSobjectExternal
 	Links DSMSobjectLinks
 }
 
+type AzureSobjectCustomMetadata struct {
+	Azure_key_state     string `json:"azure-key-state"`
+	Azure_key_name       string `json:"azure-key-name"`
```

Comment:
nit: the formatting here is inconsistent

Reply:
Modified accordingly.